### PR TITLE
Support for Support Enquries endpoint

### PR DESCRIPTION
**Pull request by bradbarrow**
### Overview

This will allow support enquiries to be constructed via the API.

### CODE REVIEW

**Comment by ags** [comment]
:+1:

File: test/unit/resources/SupportEnquiries_test.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import SupportEnquiries from "../../../src/resources/SupportEnquiries";
+
+describe("SupportEnquiries", () => {
+  describe("create", () => {
+    it("calls POST /support-enquiries with the given parameters", () => {
+      const post = sinon.spy();
+      const client = { post };
+      const supportEnquiries = new SupportEnquiries(client);
+      const params = {
+        message: "Yes this is Mc Lovin’ speaking."
```
Comment (jamesmoriarty) [inline test/unit/resources/SupportEnquiries_test.js]: :smile:

**Comment by iamliamnorton** [comment]
Does this deal with non-logged in users?

**Comment by bradbarrow** [comment]
@iamliamnorton no, the current site does. Worst case scenario we get spammed. It's fairly easy to add that in and probably will in due course. @ags mentioned rate limiting the API in general in the near future

**Comment by iamliamnorton** [comment]
Okay thats fine. We currently require an email address for non-logged in users, I'd guess would still want that then?

**Pull request merged**